src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,168 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    public static class Node<T> {
+        private Node<T> prevNode;
+        private Node<T> nextNode;
+        private T value;
+
+        public Node(T value) {
+            this.value = value;
+        }
```

Comment:
If you will make constructor not only for value, maybe code looks better? (prevNode, nextNode)